### ISSUE TITLE
mount disk as volume rather than create a volume which deletes the  data

### ISSUE THREAD

**Issue by Greylinux** [johnnyq/simpnas#52]
Hi johnny,

one feature that would be great would be to be able to mount a  disk as a volume without it deleting the data first , I know that sometimes this can cause problems depending on disk format and content so a warning could be displayed to suggest that the disk might need to be wiped if issues occur. 
I had a disk with movies on that when added, deleted the data and I needed to re upload the data after the volume was created so being able to mount the disk without wiping data first would be great.

**Comment by johnnyq** [comment]
Good Idea @GreyMarvel This is something I would love to implement but it will have to wait until after the first stable release. A heavy portion of the back end logic will have to be modified.

**Comment by johnnyq** [comment]
@GreyMarvel Im sorry to hear about your movies, i'll add a warning when creating a volume on a disk. This would be extremely useful.

**Comment by Greylinux** [comment]
> @GreyMarvel Im sorry to hear about your movies, i'll add a warning when creating a volume on a disk. This would be extremely useful.

Don't worry about the film's I always create a backup of everything before I implement anything new . So it was trivial to re upload the movies . I'm thinking more of the next person who tries simNAS .

Thanks Johnny 😊
